themes/socialbase/components/04-organisms/comment/comment.scss

```diff
@@ -40,6 +40,8 @@ a[id^="comment-"] {
 
 .comment__content {
   flex: 1;
+  width: 100%;
```

Comment:
This should be the default right? Does it break if we remove this line?

Reply:
@Kingdutch, you are right when I removed this style the changes work good. I added this style for hedging😊

Reply:
No need to hedge :) Block layout elements (like div) are 100% width by default.

Let’s remove this line and then it can be merged when tests pass :)

Reply:
@Kingdutch, I removed width style :)